src/Sulu/Bundle/PageBundle/Sitemap/PagesSitemapProvider.php

```diff
@@ -42,14 +44,21 @@ class PagesSitemapProvider extends AbstractSitemapProvider
      */
     private $environment;
 
+    /**
+     * @var ?AccessControlManagerInterface
```

Comment:
It is already sometimes used in our codebase, and even in phpstan's codebase, so it should understand it. Apart from that I would prefer that way of writing it, because it is shorter and also supported by PHP itself (other than the `|null` approach, which does not work in PHP currently).